code/modules/mob/living/simple_animal/hostile/giant_spider.dm

```diff
@@ -611,6 +561,34 @@
 		to_chat(ghost, "[link] [my_message]")
 	usr.log_talk(message, LOG_SAY, tag="spider command")
 
+/**
+ * # Web shot
+ *
+ * A projectile shot by spiders, doesn't deal damage but instead grants debuffs to those hit
+ */
+/obj/projectile/spider
```

Comment:
Rename this to `/obj/projectile/webspit` or something--this implies the projectile is the spider. Spooky.